WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsPresentingViewController.swift

```diff
@@ -3,18 +3,32 @@ import UIKit
 
 final class CardReaderSettingsPresentingViewController: UIViewController {
 
+    @IBOutlet weak var temporaryLabel: UILabel!
+
     /// An array of viewModels and related view classes
-    private var viewModelsAndViews = [CardReaderSettingsViewModelAndView]()
+    private var viewModelsAndViews: CardReaderSettingsPrioritizedViewModelsProvider?
 
     /// Set our dependencies
-    func configure(viewModelsAndViews: [CardReaderSettingsViewModelAndView]) {
+    func configure(viewModelsAndViews: CardReaderSettingsPrioritizedViewModelsProvider) {
         self.viewModelsAndViews = viewModelsAndViews
+        self.viewModelsAndViews?.onPriorityChanged = onViewModelsPriorityChange
     }
 
     override func viewDidLoad() {
         super.viewDidLoad()
         configureBackground()
         configureNavigation()
+        configureInitialState()
+    }
+
+    private func configureInitialState() {
+        onViewModelsPriorityChange(viewModelAndView: viewModelsAndViews?.priorityViewModelAndView)
+    }
+
+    private func onViewModelsPriorityChange(viewModelAndView: CardReaderSettingsViewModelAndView?) {
+        // For now, just update the label with the view identifer we should display
+        // A later PR will actually add presented views
+        temporaryLabel.text = viewModelAndView?.viewIdentifier ?? "Loading"
```

Comment:
This "loading" string is not localized because this is temporary and will be removed in an upcoming PR